### PR TITLE
Fix polarization

### PR DESCRIPTION
**Pull request by LeilyR**
renamed compartmentPolarization to compartmentalization.
Fixed couple of bugs for this module.

### CODE REVIEW

File: hicexplorer/hicAdjustMatrix.py [hunk 1]
```diff
@@ -111,19 +111,17 @@ def adjustMatrix(pArgs):
         # log.debug('genomic_regions {}'.format(genomic_regions))
         matrix_indices_regions = []
         for region in genomic_regions:
-            _regionBinRange = hic_matrix.getRegionBinRange(region[0], region[1], region[2])
+            _regionBinRange = hic_matrix.getRegionBinRange(region[0],
```
Comment (joachimwolff) [inline hicexplorer/hicAdjustMatrix.py]: Please have it in one line and change your linter and autolint settings to the rules we have defined in travis.yaml

File: hicexplorer/hicCompartmentalization.py [hunk 2]
```diff
@@ -72,36 +82,49 @@ def parse_arguments():
...
 
-def count_interactions(obs_exp, pc1, quantiles_number):
+def count_interactions(obs_exp, pc1, quantiles_number, offset):
     "Counts the total interaction on obs_exp matrix per quantile and "
     "normalizes it by the number of bins per quantile."
     chromosomes = pc1["chr"].unique()
-    normalised_sum_per_quantile = np.zeros(
-        (quantiles_number, quantiles_number))
-
+    sum = np.zeros((quantiles_number, quantiles_number))
```
Comment (joachimwolff) [inline hicexplorer/hicCompartmentalization.py]: 'sum' is a python keyword, please use another variable name.

File: hicexplorer/hicAdjustMatrix.py (hunk 1 above)
Comment (joachimwolff) [reply hicexplorer/hicAdjustMatrix.py]: Please move the changes of hicAdjustMatrix to its own PR and add a test case which shows why this change is necessary.

Comment (LeilyR) [reply hicexplorer/hicAdjustMatrix.py]: could you please check why the test from chic background model fails?

**Pull request merged**